docs/deploy/getting-started.md

```diff
@@ -0,0 +1,66 @@
+# Deploy your application on Firebase Hosting
+
+In this guide, we'll look at how to use `@angular/fire` to automatically deploy an Angular application to Firebase hosting by using the Angular CLI.
+
+## Step 1: add `@angular/fire` to your project
+
+First, you need to add the `@angular/fire` package to your project. In your Angular CLI project run:
+
+```shell
+ng add @angular/fire
+```
+
+*Note that the command above assumes you have global Angular CLI installed. To install Angular CLI globally run `npm i -g @angular/cli`.*
+
+The command above will trigger the `@angular/fire` `ng-add` schematics. Once they install `@angular/fire`. If you are not authenticated, the schematics will open a browser which will guide you through the authentication flow. Once you authenticate, you'll see a prompt to select a Firebase hosting project.
```

Comment:
When I ran ng-add, the "choose a project" prompt wasn't in alphabetical order. The Firebase CLI returns them in alphabetical, so that was unexpected for me. Probably not a huge deal since most people don't have tons of projects, but worth fixing if it isn't a ton of work.

Reply:
For convenience, we provide fuzzy search. Maybe alphabetical order will be an additional simplification. We can add this.